docs/02.API-REFERENCE.md

```diff
@@ -2412,6 +2424,81 @@ main (void)
 - [jerry_container_type_t](#jerry_container_type_t)
 
 
+## jerry_container_operation
+
+**Summary**
+
+Perform container operation on the given operands (add, delete, set, etc.).
+
+*Note*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+- This API function depends on a build option (`JERRY_BUILTIN_CONTAINER`) and can be checked
+  runtime with the `JERRY_FEATURE_MAP` , `JERRY_FEATURE_SET` , `JERRY_FEATURE_WEAKMAP` , `JERRY_FEATURE_WEAKSET`
+  feature enum values.
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The es.next profile enables this by default.
+
+*New in version [[NEXT_RELEASE]]*.
```

Comment:
This "new version" is not at the correct place, It should be just before the example "block". (Just noticed in other api descriptions this is also at an incorrect place, but that should be a different PR fix)